malada/providers/supercell.py

```diff
@@ -48,17 +48,39 @@ def provide(self, provider_path, cif_file):
             super_cell = ase.build.make_supercell(primitive_cell,
                                                   transformation_matrix)
             super_cell = self.get_compressed_cell(super_cell,
-                                                  self.parameters.compression)
+                                                  compression_factor = self.parameters.compression,
+                                                  density = self.parameters.mass_density,
+                                                  radius = self.parameters.WS_radius)
             ase.io.write(self.supercell_file,
                          super_cell, format="vasp", long_format=True)
         else:
             copyfile(self.external_supercell_file, self.supercell_file)
             print("Getting <<supercell>>.vasp file from disc.")
 
     @staticmethod
-    def get_compressed_cell(supercell, compression_factor):
-        supercell.set_cell(supercell.get_cell() * compression_factor,
-                           scale_atoms=True)
+    def get_compressed_cell(
+        supercell,
+        compression_factor=None,
+        density=None,
+        radius=None,
+        units_density="g_cm^3",
```

Comment:
This PR looks good to me, just one minor thing: In MALA we have units as "g/(cm^3)", instead of "_" - I know that this was not your implementation choice, I made the inconsistency myself. But could you maybe address this in this PR as well, since it touches the relevant files? Thanks!

Reply:
Done